src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -238,20 +249,22 @@ const CancelRequestButton = ({
           </Text>
         </MenuDropdownItem>
       </MenuDropdownButton>
-      <CancelRequestModal isOpen={isOpen} onClose={onClose} />
+      <CancelRequestModal
+        isOpen={isOpen}
+        onClose={() => {
+          setIsApproved(false)
+          onClose()
+        }}
+      />
     </>
   )
 }
 
 // NOTE: Utility component exists to soothe over state management
```

Comment:
is this comment still relevant?

Reply:
Good catch, removed in https://github.com/isomerpages/isomercms-frontend/pull/1263/commits/414b75d87d6d4472ae4fc30ba514356cbe90c3fb